include/libm2k/contextbuilder.hpp

```diff
@@ -30,6 +30,7 @@
 enum ContextTypes {
 	CtxFMCOMMS,
 	CtxM2K,
+        CtxLIDAR,
```

Comment:
this should be indented with tabs;
it's indented with spaces;

when adding new code, it's good to preserve the original styling and spacing

Reply:
Done